python/cudnn/grouped_gemm/grouped_gemm_quant/moe_blockscaled_grouped_gemm_quant_rubin.py

```
@@ -0,0 +1,2519 @@
+                b_template_stride = (c1_64, b_stride_size, c0)
+            b_template_layout = cute.make_layout((n, k, cutlass.Int32(1)), stride=b_template_stride)
+            b_ptr_typed = cute.make_ptr(self.b_dtype, b.toint(), AddressSpace.gmem, assumed_align=16)
+            b = cute.make_tensor(b_ptr_typed, b_template_layout)
+
+            sfb_ptr_typed = cute.make_ptr(self.sf_dtype, sfb.toint(), AddressSpace.gmem, assumed_align=16)
+            sfb_layout = blockscaled_utils.tile_atom_to_shape_SF((n, k, cutlass.Int32(1)), self.sf_vec_size)
+            sfb = cute.make_tensor(sfb_ptr_typed, sfb_layout)
+
+        # ---- SFD setup ----
+        self.generate_sfd = sfd_row_tensor is not None and norm_const_tensor is not None
+        if cutlass.const_expr(self.generate_sfd == False):
```

Ruff (0.16.0)

[error] Avoid equality comparisons to `False`; use `not self.generate_sfd:` for false checks

Replace with `not self.generate_sfd`

(E712)